src/css/_tooltip.css

```diff
@@ -0,0 +1,55 @@
+/* adapted from https://popper.js.org/docs/v2/tutorial/ */
+
+/* how to use: Create `icon` element and `popup` element in html
+
+`popup` needs to be the direct next sibling of `icon` and has class tooltip
+
+.tooltip class in this file decides how the popup looks like
```

Comment:
@kobexamoh I think you should work on this file to improve the looks of the popup.